mentoris/views.py

```diff
@@ -56,3 +57,21 @@ def user_info(request, user_id):
     return render(
         request, "mentapp/profile.html", {"user_profile": user_profile, "email": email}
     )
+
+
+def request_translation(request, user_id):
+    # need to verify email to ses when they sign up in order for this to work
+    email = get_object_or_404(Email, user_id=user_id)
```

Comment:
Figured this out today during my sign_up page implementation: We need to make sure we are accessing the email where is_primary = True. Otherwise, if the user has multiple email addresses, then a list will get returned instead of a single object.

Reply:
Thanks for the heads up!